draft-ietf-quic-multipath.md

```diff
@@ -711,6 +711,26 @@ delay will most of the time relate to the path with the shortest latency.
 To collect ACK delays on all the paths, hosts should rely on time stamps
 as described in {{QUIC-Timestamp}}.
 
+If one chooses not to use time-stamps but wants to get reasonable estimation of
+RTTs on multiple paths with one packet number space, the following practices
+can be used:
+
+* The receiver side counts the number of ACK-eliciting packets received for each paths,
+  and keeps a per-path ACK timer. An ACK from that path is triggered when the
+  number of ACK-eliciting packets received on that path surpasses the path's
+  ACK-eliciting threshold or the path's ACK-timer expires.
+
+* Each path records the largest packet received on that path and the receive
+  time of that packet. When an ACK is sent on that path, the ACK Delay field is
+  re-calculated using that path's largest packet receive time.
```

Comment:
It sounds odd to me to change the meaning of the ACK Delay field for a particular use case...